python/lsst/ts/mtaos/model.py

```diff
@@ -1217,9 +1219,7 @@ async def _poll_butler_outputs(
         start_time = time.time()
         elapsed_time = 0.0
 
-        pair_id = extra_id if extra_id is not None else intra_id
```

Comment:
I would suggest just leave `pair_id` alone here, replacing `extra_id` and `intra_id` with their `self` equivalents.